src/ros_tcp_endpoint/server.py

```diff
@@ -96,6 +97,9 @@ def send_unity_message(self, topic, message):
     def send_unity_service(self, topic, service_class, request):
         return self.unity_tcp_sender.send_unity_service(topic, service_class, request)
 
+    def topic_list(self, data):
+        return RosUnityTopicListResponse(self.source_destination_dict.keys())
```

Comment:
Not the best example but wouldn't the source dest dict below be valid?

```
tcp_server.start({
    'robot_pos_rot': RosPublisher('pos_rot', PosRot, queue_size=10),
    'robot_pos_rot_mapper': RosSubscriber('pos_rot', PosRot, tcp_server)
})
```

In that case it might be better to do something like,

```
[x.topic for x in self.source_destination_dict.keys() ]
```

Reply:
Uh... if that's supposed to be valid (and I'm not sure it is), I don't understand why we're using a dict at all?